webpack/config.js

```diff
@@ -170,11 +170,18 @@ module.exports = {
         loader: 'url-loader?limit=8192'
       },
       {
-        test: /\.(png|svg|jpg|gif)$/i,
+        test: /\.(svg)$/i,
         loaders: [
           'url-loader?limit=8192',
           'image-webpack?optimizationLevel=7&interlaced=false'
         ]
+      },
+      {
+        test: /\.(jpe?g|png|gif|svg)$/i,
+        loaders: [
+          'file?hash=sha512&digest=hex&name=[hash].[ext]',
+          'image-webpack?bypassOnDebug&optimizationLevel=7&interlaced=false'
+        ]
```

Comment:
What’s the rationale for this split? I suspect what we  might _actually_ wanna change is the `url-loader` config, as it passes through to `file-loader` for files larger than `limit`, and I suspect ours is too big!